py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -368,6 +368,8 @@ def __init__(self, remote_server_addr, keep_alive=False, resolve_ip=True):
                 ('POST', '/session/$sessionId/window/rect'),
             Command.GET_WINDOW_RECT:
                 ('GET', '/session/$sessionId/window/rect'),
+            Command.W3C_MINIMIZE_WINDOW:
```

Comment:
THX for your review.
Please help to check the commit I closed earlier.
https://github.com/SeleniumHQ/selenium/pull/4272
And I already push commits for minimize command in webdriver-rust part which already merged, geckodriver & marionette part which are under reviewing.
The PR#4272 is just like the implementation of the maximize command.
THX again!